config/locales/en.yml

```diff
@@ -30,24 +30,66 @@
 # available at https://guides.rubyonrails.org/i18n.html.
 
 en:
+  activemodel:
+    errors:
+      models:
+        user:
+          attributes:
+            current_password:
+              blank: Enter your current password.
+            first_name:
+              blank: Enter a first name.
+            last_name:
+              blank: Enter a surname.
+            postcode:
+              blank: Enter your setting's postcode.
+            email:
+              blank: Enter an email address.
+            password:
+              blank: Enter a password.
+              too_short: Password is too short.
+            password_confirmation:
+              confirmation: Confirmation doesn't match password.
   activerecord:
     errors:
       models:
         user:
           attributes:
             current_password:
-              blank: Current password cannot be blank if changing.
+              blank: Enter your current password.
+            first_name:
+              blank: Enter a first name.
+            last_name:
+              blank: Enter a surname.
+            postcode:
+              blank: Enter your setting's postcode.
             email:
-              blank: Email address must not be blank.
+              blank: Enter an email address.
             password:
-              blank: Password must not be blank.
+              blank: Enter a password.
               too_short: Password is too short.
             password_confirmation:
               confirmation: Confirmation doesn't match password.
+  phase_banner: "This is a new service, your %{feedback_url} will help us improve it.)"
```

Comment:
I know this is pedantic but .... it is not a URL that is being passed in. It is a link (that is, an HTML anchor with a URL within. Therefore I think `feedback_url` should be renamed `feedback_link`